discord/ext/commands/errors.py

```diff
@@ -202,6 +213,186 @@ class NotOwner(CheckFailure):
     """
     pass
 
+class MemberNotFound(BadArgument):
+    """Exception raised when the member provided was not found in the bot's
+    cache.
+
+    This inherits from :exc:`BadArgument`
+
+    .. versionadded:: 1.5
+
+    Attributes
+    -----------
+    member: :class:`str`
+        The member supplied by the caller that was not found
+    """
+    def __init__(self, member):
+        self.member = member
```

Comment:
For the sake of custom converters and error handlers which may raise / handle these exceptions.
Could it be wise to rather than uniquely name these attributes based on their relevant argument type, use a more generic Error.argument attribute.

conscider this example:
```py
@bot.event
async def on_command_error(ctx: Context, error: Exception):
    if isinstance(error, (MemberNotFound, UserNotFound)):
        if isinstance(error, MemberNotFound):
            argument = error.member
        elif isinstance(error, UserNotFound):
            argument = error.user
        bot.log.info(f'Converter for command {ctx.command.name} failed, argument supplied: "{argument}"')
```

having all errors share a common attribute name would remove the need for handling individual cases like shown.

Reply:
Done, renamed the argument in all converter error classes to `arg`